Makefile

```diff
@@ -20,6 +20,9 @@ distclean: clean
 
 -include config.mk
 
+setup.ml: _oasis
+	@oasis setup
+
```

Comment:
I'm not sure I'd call it standard :-) but I've been doing it recently and
it's been ok.

In mirage-block-volume I added a "make release" target. I then do:

```
git checkout -b release.1.0
make release
git add <autogen crap>
...
```

My goal is to keep the master branch clean and only check in the stuff so
it appears in the release tarballs so I don't have to make it a dependency
in opam-repository.



On Fri, Mar 20, 2015 at 5:49 PM, Thomas Gazagnaire <notifications@github.com
> wrote:

> In Makefile
> <https://github.com/djs55/message-switch/pull/18#discussion_r26860658>:
>
> > @@ -20,6 +20,9 @@ distclean: clean
> >
> >  -include config.mk
> >
> > +setup.ml: _oasis
> > +	@oasis setup
> > +
>
> Is that standard? I quite like to do the same thing for my projects... I
> get tired of that autogen crap.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/djs55/message-switch/pull/18/files#r26860658>.
>



-- 
Dave Scott